src/index.ts

```diff
@@ -10,16 +11,45 @@ module.exports = (app) => {
     const response = await axios.get(telegram_bot_link,{headers:headers});
     //app.log.info(response);
     await context.github.issues.createComment(issueComment);
+    }
   });
   app.on('issue_comment.created',async (context) => {
     if(context.payload.sender.type != "Bot"){
+      let commentBody = context.payload.comment.body;
+      if (commentBody.startsWith("EDIT")){
```

Comment:
Yes its really nice you are doing this but i guess making a function which maps a json with data points to the format we need and vice versa then it would be changing that one varible and things would look neater i guess..... let me know what you have thought

Reply:
and this thing i guess we could explicit ly use probot commands  
Reference : 
- [Probot-commands repo](https://github.com/probot/commands)
- [Probot docs extentions](https://probot.github.io/docs/extensions/)

I believe doing this is better then one will have a good idea about this feature and use it in cops bot in a better way